specification/maps/data-plane/Microsoft.Maps/DEM/preview/1.0/elevation.json

```diff
@@ -0,0 +1,631 @@
+            "description": "The samples parameter specifies the number of equally spaced points at which elevation values  should be provided along a polyline path. The number of samples should range from 2 to 2,000. Default value is 10 if not provided.",
+            "type": "number",
+            "minimum": 2,
+            "maximum": 2000,
+            "default": 10,
+            "required": false
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/LinesResult"
+            }
+          },
+          "400": {
+            "$ref": "#/responses/400"
+          },
+          "401": {
+            "$ref": "#/responses/401"
+          },
+          "403": {
+            "$ref": "#/responses/403"
+          },
+          "404": {
+            "$ref": "#/responses/404"
+          },
+          "500": {
+            "$ref": "#/responses/500"
+          }
+        }
+      }
+    },
+    "/elevation/lattice/{format}": {
+      "get": {
+        "summary": "Get elevation data at equally-spaced locations within a bounding box.",
+        "description": "**Applies to**: S1 pricing tier.\n\nThe Get Data for Bounding Box API provides elevation data at equally-spaced locations within a bounding box.  A bounding box is defined by the coordinates for two corners (southwest, northeast) and then subsequently  divided into rows and columns. Elevations are returned for the vertices of the grid created by the rows and  columns. Up to 2000 elevations can be returned in a single request. The returned elevation values are ordered,  starting at the southwest corner, and then proceeding west to east along the row. At the end of the row,  it moves north to the next row, and repeats the process until it reaches the far northeast corner.",
+        "operationId": "Elevation_GetDataForBoundingBox",
+        "x-ms-examples": {
+          "GetDataForBoundingBox": {
+            "$ref": "./examples/GetDataForBoundingBox.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionKey"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/JsonFormat"
+          },
+          {
+            "name": "bounds",
+            "in": "query",
+            "description": "The string that represents the rectangular area of a bounding box. The value of the bounds parameter  must be an array of length 4, with WGS84 longitude and latitude of the southwest corner followed by  WGS84 longitude and latitude of the northeast corner. The string is presented in the following  format: [SouthwestCorner_Longitude, SouthwestCorner_Latitude, NortheastCorner_Longitude,  NortheastCorner_Latitude].",
```

Comment:
This description is not clear to a user of the REST API. Arrays are not generally part of the "type system" for a query parameter. I'd say that you can specify one or more values (and is specifying a single value not allowed using this API?) and how to specify multiple values and not mention array in the description.

Reply:
Done.  Specified the four values comma separated and leave "array" out of the description.